### ISSUE TITLE
Composer Install on Volume not working

### ISSUE THREAD

**Issue by willcode4food** [wodby/docker4drupal#62]
Hello,

I am trying to use composer to install Solarium for use with Solr and Search API.  Using composer to install Solarium results in an error  rmdir .... Directory not empty

I think i have it narrowed down to an issue with the volume on the Docker container.  (See: http://stackoverflow.com/questions/14630199/symfony2-error-when-installing-via-composer)  

Any ideas on how to resolve/workaround this issues?

EDIT: composer update also fails with the same type of error

**Comment by PavelPrischepa** [comment]
Could you provide more info like commands, output, docker-compose file?

**Comment by willcode4food** [comment]
Yes.  

docker-compose services for PHP container
```
php:
    image: wodby/drupal-php:7.0 # Allowed: 7.0, 5.6.
    environment:
      PHP_SITE_NAME: dev
      PHP_HOST_NAME: localhost:8000
#      PHP_DOCROOT: public # Relative path inside the /var/www/html/ directory.
      PHP_SENDMAIL_PATH: /usr/sbin/sendmail -t -i -S mailhog:1025
      # PHP_XDEBUG_ENABLED: 0
      # PHP_XDEBUG_AUTOSTART: 0
      # PHP_XDEBUG_REMOTE_CONNECT_BACK: 0         # This is needed to respect remote.host setting bellow
      # PHP_XDEBUG_REMOTE_HOST: "10.254.254.254"  # You will also need to 'sudo ifconfig lo0 alias 10.254.254.254'
    volumes:
      - ./:/var/www/html
```

Terminal entry into docker container using the following command
`docker exec -it -u 82 <container name or hash>`

Command to install solarium package
`composer update`

Error Message
```
> Drupal\Core\Composer\Composer::vendorTestCodeCleanup
Script Drupal\Core\Composer\Composer::vendorTestCodeCleanup handling the post-package-update event terminated with an exception


  [ErrorException]
  rmdir(/var/www/html/vendor/symfony/console/Tests/Fixtures): Directory not empty
```

**Comment by csandanov** [comment]
What OS do you have on your host machine? Linux/macOS? What happens when you execute `docker-compose exec php composer update`?

**Comment by willcode4food** [comment]
You bring up a good point.  I am running windows 10.  Also `compose exec` does not work with composer on windows.

**Comment by csandanov** [comment]
We've released all-new version 2.0.0, look up test/ dir to see how we test it with Apache Solr (we also use composer). In other words, we use https://github.com/drupal-composer/drupal-project and make composer required drupal/search_api_solr and it handles search_api dependencies automatically
